packages/mdc-shape/_functions.scss

```diff
@@ -143,6 +124,61 @@
   }
 }
 
+//
+// Accepts radius number or list of 2 radius values and returns 4 value list with
+// top-left value, top-right value, bottom-right 0, bottom-left 0.
+// Throws error when length of radius is more than 2.
+//
+// Example:
+//
+// 1. mdc-shape-mask-top-radius((2px, 3px)) => 2px 3px 0 0
+// 2. mdc-shape-mask-top-radius(8px) => 8px 8px 0 0
+//
+@function mdc-shape-mask-top-radius($radius) {
```

Comment:
Can this be simplified to just reuse `mdc-shape-mask-radius($radius, 1 1 0 0)` now?

Should we even just remove this and always use mask-radius?

Reply:
Removed `mask-top-radius` mixin in next PR.